anyric/super-record
# #167133622 Implement a navigation functionality

#### What does this PR do?
This PR implement a navigation functionality for the application.

#### Description of Task to be completed?
- Add a navigation bar to the dashboard

#### How should this be manually tested?
- Clone this branch
- In the root directory run `python manage.py runserver`
- Navigate to `http://127.0.0.1:8000/login` to see the nav bar when not login
- Login to see the nav bar when logged in

#### Any background context you want to provide?
Currently the application dashboard doesn't have a navigation bar hence making it difficult to navigate the pages.

#### What are the relevant pivotal tracker stories?
[#167133622](https://www.pivotaltracker.com/story/show/167133622)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29800415/60975683-0856a100-a335-11e9-9d43-06aa361b639b.png)

#### Questions:
N/A
